README.md

```diff
@@ -51,6 +51,10 @@ Copy `config/config-example.js` into `config/config.js`, and edit as you please.
 
 Congratulations, you're done setting up Pokémon Showdown.
 
+After you will need to install the dependencies:
+
+    npm install
```

Comment:
That's probably specific to the Windows code. Why can't the Windows code auto-install? @Morfent?